padertorch/train/hooks.py

```diff
@@ -223,7 +223,7 @@ def update_summary(self, review):
             self.summary['histograms'][key] = \
                 self.summary['histograms'][key][-1000000:]
         for key, buffer in popped_review.pop('buffers', dict()).items():
-            self.summary['buffers'][key].extend(self._detach(buffer))
+            self.summary['buffers'][key].extend(self._to_list(self._detach(buffer)))
```

Comment:
@TCord Why is here `extend` used? I would say it should be `append`. We should make no assumptions about the buffer and don't manipulate the value (except detach). So don't call `_to_list`. For other entries `_to_list` is fine, because the value will be prepared for tensorboardX

Reply:
I used `extend` to make it similar to the other summary types, but forgot that it inherently makes assumptions on the data again. I also think, `append` would be the better choice here.

Reply:
changed it